### PR TITLE
WIP: Basic implementation

### PR DESCRIPTION
**Pull request by angelbarrera92**
I am still working on it. 
I'll enable the pipeline once I've got  some make target passing (currently lint does not pass)

### CODE REVIEW

File: pkg/checkresult/v1beta1/v1beta1.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	}
+
+	return &CheckResultV1Beta1Client{
+		kubeclient: *kc,
+		mapper:     mapper,
+		mapping:    *mapping,
+	}, nil
+}
+
+func (c *CheckResultV1Beta1Client) CheckResults(namespace string) CheckResultInterface {
```
Comment (nandajavarma) [inline pkg/checkresult/v1beta1/v1beta1.go:66]: Why is it that we are hard-binding to a namespace here?

Comment (angelbarrera92) [reply pkg/checkresult/v1beta1/v1beta1.go:66]: Because the CRD that we have created is namespace scoped.

Comment (angelbarrera92) [reply pkg/checkresult/v1beta1/v1beta1.go:66]: but i think there is a missing piece of code here. Checking

Comment (nandajavarma) [reply pkg/checkresult/v1beta1/v1beta1.go:66]: thanks for the clarity!

**Review by nandajavarma — APPROVED** [review]
Overall looks really good! :)

**Pull request merged**